README.md

```diff
@@ -789,6 +790,10 @@ Supported `temp` values: 1-65279 mired. Actual color temperature supported by en
 
 - Mix of warm white intensity and cold white intensity forms color temperature.
 
+- Duration property within cmd.color.start_transition command is optional and means seconds.
+
+- For Z-Wave protocol, slave devices will accept only values from range 1...127 seconds or 1...127 minutes. This is why sup_durations property was introduced. Zwave adapter will be responsible for converting seconds above value 127 to minutes.
```

Comment:
MAYBE more formal:

- In the Z-Wave protocol color component transition duration can only be set between 1 - 127 seconds or 1 - 127 minutes. In order to fully support the Z-Wave protocol, sup_durations property can contain an array of ranges.

Reply:
done